src/Tasks/Copy.cs

```diff
@@ -262,7 +262,9 @@ private void LogDiagnostic(string message, params object[] messageArgs)
             if (!linkCreated)
             {
                 // Do not log a fake command line as well, as it's superfluous, and also potentially expensive
-                Log.LogMessageFromResources(MessageImportance.Normal, "Copy.FileComment", sourceFileState.Name, destinationFileState.Name);
+                string sourceFilePath = Path.GetFullPath(sourceFileState.Name);
+                string destinationFilePath = Path.GetFullPath(destinationFileState.Name);
```

Comment:
Should these instead use `FileUtilities.GetFullPathNoThrow`? I think it's less allocatey on Windows and it's used in `GenerateResource` so it should be available in this assembly already.

Reply:
```FileUtilities.GetFullPathNoThrow``` is a little misleading since it calls ```FileUtilities.NormalizePath``` which replaces the directory separators on non-Windows. Not sure if its callers depend on that behavior.

Reply:
Is that a problem in this situation though?

Reply:
Path would be semantically wrong in the log but that's really it

Reply:
Oops completely misread what ```FixFilePath``` does, it shouldn't be a problem